### DISCUSSION TITLE
write out constexpr variable within object

### DISCUSSION THREAD

**Discussion by jbbjarnason** [stephenberry/glaze#315] [Q&A] [answered]
Consider the following example:
```C++
struct test_case {
static constexpr std::string_view info{ "information" };
static constexpr auto index{ 42 };
std::string value;
};
template<>
glz::meta<test_case> {
static constexpr auto value{ glz::object("information", test_case::info, "index", test_case::index, "value", &test_case::value) };
};
```
This actually compiles(to my best knowledge) but does not write constexpr variables to json.
It would be really nice being able to store compile time metadata which would be written out but ofcourse would not be read.

Is there a way to achieve this?

**Comment by stephenberry** [comment, marked as answer]
Thanks for asking. I've wanted this behavior as well, so I'll create an issue for it.

**Reply by stephenberry** [reply]
@jbbjarnason This is now supported as of #316
